.github/workflows/go.yml

```
@@ -142,8 +142,9 @@ jobs:
           #- 1.14.x
           #- 1.15.x
           #- 1.16.x
-          - 1.17.x
-          - 1.18.x
+          #- 1.17.x
+          #- 1.18.x
```

```suggestion
          - 1.18.x
```

always keep the two last major versions, except if it's not possible